openedx/core/djangoapps/user_authn/tests/test_cookies.py

```diff
@@ -33,13 +33,13 @@ def _get_stub_session(self, max_age=604800):
     def _get_expected_header_urls(self):
         expected_header_urls = {
             'logout': reverse('logout'),
-            'resume_block': retrieve_last_sitewide_block_completed(self.user)
+            'resume_block': retrieve_last_sitewide_block_completed(self.user),
+            'account_settings': reverse('account_settings'),
```

Comment:
Reviewer note: This change makes the url available in Studio, which is why this test was updated.